src/server.cpp

```diff
@@ -3181,14 +3181,14 @@ std::string Server::getStatusString()
 	bool first = true;
 	os << " | clients: ";
 	if (m_env) {
-		std::vector<session_t> clients = m_clients.getClientIDs();
-		for (session_t client_id : clients) {
-			RemotePlayer *player = m_env->getPlayer(client_id);
+		// Get the list of player names
```

Comment:
comments like this can go, the following line of code is self explaining

Reply:
```suggestion
```